### PR TITLE
Check if is safe to update when user click "update" in UI

### PR DESCRIPTION
**Pull request by fernandog**
@miigotu ok now?

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: sickbeard/versionChecker.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def run(self, force=False):
             sickbeard.BRANCH = self.get_branch()
 
             if self.check_for_new_version(force):
-                if sickbeard.AUTO_UPDATE:
+                if sickbeard.AUTO_UPDATE or force is True:
```
Comment (miigotu) [inline sickbeard/versionChecker.py]: why `is True` ? force is only True or False you dont need an extra comparison.

**Comment by OmgImAlexis** [comment]
@miigotu why was this merged when it just adds duplicate code?

Doesn't `if self.check_for_new_version(force):` check if it's true?

**Comment by fernandog** [comment]
@OmgImAlexis manual update (green div) used to call "run_backup" directly without doing the checks auto-update does

**Comment by miigotu** [comment]
@OmgImAlexis because it gets way to exhausting arguing or explaining for hours on end, only to move on to the next thing and do the same arguing and explaining again.

**Pull request merged**